sdk/js/src/Request.ts

```diff
@@ -93,8 +93,9 @@ export const getConfigJson = async (
         return res.data
     } catch (ex: any) {
         logger.error(`Request to get config failed for url: ${url}, ` +
-            `response message: ${ex.message}, response data: ${ex?.response?.data}`)
-        throw new Error('Failed to download DevCycle config.')
+            `response message: ${ex.message}, response data: ${JSON.stringify(ex?.response?.data.data.errors)}`)
+        throw new Error(`Failed to download DevCycle config. ` +
+            `Error details: ${JSON.stringify(ex?.response?.data.data.errors)}`)
```

Comment:
just a nitpick, but you could store this to a variable so that we don't need to duplicate stringifying the errors

Reply:
will update, makes sense :)